.github/workflows/cflite.yml

```diff
@@ -4,6 +4,10 @@ on:
   pull_request:
     paths:
       - '**'
+# cancel any previous runs on the same PR
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
```

Comment:
can you add new line after, just a nit pick.

Reply:
reformatted the file to have new lines between top-level keys like the other files @ 347f16f